packages/plugins/plugin-braze-middleware/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@segment/analytics-react-native-plugin-braze-middleware",
+  "version": "0.1.0",
+  "description": "The hassle-free way to add Segment analytics to your React-Native app.",
+  "main": "lib/commonjs/index",
+  "scripts": {
+    "build": "bob build",
+    "test": "jest",
+    "typescript": "tsc --noEmit",
+    "clean": "rimraf lib node_modules",
+    "release": "semantic-release"
+  },
+  "keywords": [
+    "segment",
+    "react-native",
+    "ios",
+    "android"
+  ],
+  "module": "lib/module/index",
+  "types": "lib/typescript/src/index.d.ts",
+  "react-native": "src/index",
+  "source": "src/index",
+  "files": [
+    "src",
+    "lib",
+    "android",
+    "ios",
+    "cpp",
+    "segment-analytics-react-native.podspec",
+    "package.json",
+    "!src/**/*.e2e.mock.js",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
```

Comment:
add also here the `!android/.gradle` and the other one that you just added in `core`